src/views/start/index.jsx

```diff
@@ -13,6 +13,12 @@ export default class Start extends Component {
     this.sortByCommits = _ => this.changeSorting(this.state.fileStats, 'commits');
     this.sortByFile = _ => this.changeSorting(this.state.fileStats, 'file');
     this.sortByDate = _ => this.changeSorting(this.state.fileStats, 'latestDate');
+    this.help = this.help.bind(this);
+    this.openFolderDialogValue = '/';
+    this.queryValue = "bugfix";
+    this.fileExtension = '*';
+    this.pathOfDirectory = __dirname;
+    this.currentPath = this.pathOfDirectory.substr(0, this.pathOfDirectory.length - 15);
```

Comment:
we can make that but please use lastIndexOf(/) to jump back some folders. Because this will stop working if we change the parent folder name